src/ir/operation_test.cc

```diff
@@ -95,17 +97,17 @@ INSTANTIATE_TEST_SUITE_P(
         OperationTestData({nullptr,
                            &OperationTest::minus_op,
                            {{"const", Attribute::Create<Int64Attribute>(10)}},
-                           {{"arg", Value(value::Any())}},
-                           "%0 = core.minus [const: 10](arg: <<ANY>>)\n"}),
+                           {Value(value::Any())},
+
+                           "%0 = core.minus [const: 10](<<ANY>>)\n"}),
         OperationTestData(
             {&OperationTest::first_block_,
              &OperationTest::minus_op,
              {{"const", Attribute::Create<Int64Attribute>(10)}},
-             {{"a", Value(value::BlockArgument(OperationTest::first_block_,
-                                               "arg0"))},
-              {"b", Value(value::BlockArgument(OperationTest::first_block_,
-                                               "arg1"))}},
-             "%0 = core.minus [const: 10](a: %0.arg0, b: %0.arg1)\n"})));
+             {Value(value::BlockArgument(OperationTest::first_block_, "arg0")),
+              Value(value::BlockArgument(OperationTest::first_block_, "arg1"))},
+             "%0 = core.minus [const: "
+             "10](%0.arg0, %0.arg1)\n"})));
```

Comment:
Looks like your string got sheared here. Might want to put it all on one line to improve readability.

Reply:
done